spec/index.bs

```diff
@@ -13992,7 +13992,9 @@ interface GPUCanvasContext {
                 1. [=?=] [$Validate texture format required features$] of
                     |configuration|.{{GPUCanvasConfiguration/format}} with |device|.{{GPUObjectBase/[[device]]}}.
                 1. [=?=] [$Validate texture format required features$] of each element of
-                    |configuration|.{{GPUTextureDescriptor/viewFormats}} with |device|.{{GPUObjectBase/[[device]]}}.
+                    |configuration|.{{GPUCanvasConfiguration/viewFormats}} with |device|.{{GPUObjectBase/[[device]]}}.
+                1. If [=Supported context formats=] does not [=set/contain=]
+                            |configuration|.{{GPUCanvasConfiguration/format}}, throw a {{TypeError}}.
```

Comment:
Just a note, in case anyone has feelings about it (I'm OK either way): this could be equivalently described by changing the type of GPUCanvasConfiguration.format to a new enum which has only 3 values and is a subset of `GPUTextureFormat`.

Reply:
There may be new formats allowed in the future so I'm guessing it's better to use `GPUTextureFormat` than some other enum.  Like `rgb10a2unorm`

Reply:
Yes would be great if we can introduce `rgb10a2unorm` at some point but outside the scope for now, so agreed with @greggman's comment

Reply:
It would have exactly the same WebIDL/JS behavior either way (a TypeError, and we can always add formats to a new enum). The practical difference I think is really in what language bindings would do - whether TypeScript would reject other values, and whether bindings to other statically-typed languages use separate enums (usually with the same numeric values).